From ea96e5684af98e28e704dc26f12b793c6032ada6 Mon Sep 17 00:00:00 2001
From: Elliott Sales de Andrade <quantum.analyst@gmail.com>
Date: Fri, 4 Jun 2021 03:20:38 -0400
Subject: [PATCH 4/4] Use new style for test_text_urls_tex.

This test does not depend on the font style, just that the annotation
appears over it. Switching to mpl20 style stops using Helvetica, and
thus avoids the issue with texlive 2021 on Fedora Rawhide.

Signed-off-by: Elliott Sales de Andrade <quantum.analyst@gmail.com>
---
 lib/matplotlib/tests/test_backend_pdf.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/matplotlib/tests/test_backend_pdf.py b/lib/matplotlib/tests/test_backend_pdf.py
index 06bfe1cdb1..548149d19d 100644
--- a/lib/matplotlib/tests/test_backend_pdf.py
+++ b/lib/matplotlib/tests/test_backend_pdf.py
@@ -238,6 +238,7 @@ def test_text_urls():
                 assert annot.Rect[1] == decimal.Decimal(y) * 72
 
 
+@pytest.mark.style('mpl20')
 @needs_usetex
 def test_text_urls_tex():
     pikepdf = pytest.importorskip('pikepdf')
-- 
2.31.1

